packages/react/src/grid/GridCell.tsx

```diff
@@ -0,0 +1,28 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+import clsx from 'clsx'
+import { PropsWithChildren } from 'react'
+import { PageGridColumnNumber } from './PageGrid'
+
+export type GridCellProps = PropsWithChildren<{
+  gridColumns?: [PageGridColumnNumber, PageGridColumnNumber]
+}>
+
+export const gridCellClassNames = (gridColumns?: [PageGridColumnNumber, PageGridColumnNumber]) => {
+  if (!gridColumns) {
```

Comment:
just a shorthand return would be enough. But not sure what the code style convention we have.
```
if (!gridColumns) return
```

Reply:
Changed it to return an empty string – I missed that this wasn’t a component but a function, and our [consistent-return](https://eslint.org/docs/latest/rules/consistent-return) rule of ESLint wants all return values to be of the same type.